File: remove_string_spaces.rb
def no_space(x)
  current_string = x
  while current_string.include?(" ")
    current_string = current_string.sub(" ", "")
  end
  return current_string
end

def no_space(x)
  x.gsub(' ', '')
end

def no_space(x)
  x.delete(' ')
end

File: removeStringSpaces.js
function noSpace(x){
  return x.split(' ').join('')
}

function noSpace(x){
  currentString = x;
  while (currentString.includes(' ')) {
    currentString = currentString.replace(" ", "");
  }
  return currentString;
}

function noSpace(x){
  currentString = x;
  while (currentString !== currentString.replace(' ', '')) {
    currentString = currentString.replace(' ', '');
  }
  return currentString;
}
